experiments.json

```diff
@@ -21,5 +21,12 @@
       "min": "0",
       "max": "100"
     }
-  }
+  },
+  "search-term": {
+    "match": {},
+    "buckets": {
+      "min": "0",
+      "max": "50"
+    }
+  },
```

Comment:
I see the travis build failed here... maybe it's caused by the trailing comma?

This makes me feel like we should have some manual test process where we test all changes on a stage branch before merging to master.